src/components/Form.js

```diff
@@ -36,104 +36,127 @@ export default function Form({
   }, [editReviewState, setValue])
   return (
     <StyledForm onSubmit={handleSubmit(onFormSubmit)}>
-      <label>
+      <Label>
```

Comment:
Is this a styled component? Maybe rename it to 'LabelStyled' to make it clear in the code. (Like you did with `StyledForm`)

Reply:
I already added a styled to each of my form components and the code looked awful, so I changed it back. I need a StyledForm because Form is already in use, but there is no need to name it StyledLabel or LabelStyled.